lib/linguist/vendor.yml

```diff
@@ -302,3 +302,6 @@
 
 # Android Google APIs
 - (^|/)\.google_apis/
+
+# Jenkins Pipeline
+- ^Jenkinsfile$
```

Comment:
Yes, that was intentional, and by analogy with very domain-specific files like `Vagrantfile` which you would not consider associated with Ruby development. Groovy is used as a host for the EDSL, but examples like [this](https://github.com/jenkinsci/pipeline-examples/blob/0a7aa6180e14fd17901a514c5e6bb356e416f41f/global-library-examples/global-function/Jenkinsfile#L5-L15) show that the actual `Jenkinsfile` can be made to look more like a config file than a program. Furthermore, allowable methods calls are restricted by a sandbox to a whitelist, and execution proceeds via a custom interpreter via continuation-passing transform which imposes various constraints on serializability etc., so the result does not look much like a general-purpose Groovy program.